core/shared/src/main/scala/fs2/Stream.scala

```diff
@@ -3089,6 +3089,21 @@ final class Stream[+F[_], +O] private[fs2] (private[fs2] val underlying: Pull[F,
   override def toString: String = "Stream(..)"
 }
 
+class StreamMonad[F[_]] extends Monad[Stream[F, *]] {
```

Comment:
Could we place this inside the `Stream` companion object and mark it as `private`?

Reply:
OK, I will do it the next day :), (it's a little late in my place)!